tox.ini

```diff
@@ -14,13 +14,25 @@ basepython = python3
 deps =
   autopep8
   flake8
+  # checks that import statements are grouped and ordered in a consistent and PEP 8-compliant way
   flake8-import-order
 commands =
   autopep8 --exit-code --aggressive --diff -r .
```

Comment:
do we want to add an `--in-place` flag to allow autopep8 to modify files?

Reply:
I was trying to test this. Does adding `--in-place` mean we need to remove `--diff`?

Reply:
Good question - seems like it. The question here would be: do we want to have these changes made for us, or do we want to review a diff and then make the changes ourselves?

HOWEVER, given my comment on #47 this may not even be an issue because this testenv may go away.

Reply:
I would lean toward having us review the diff and make changes ourselves at this stage to be more in control and understand what is changing, with the idea that we can change/revisit that at any point in the future. I'll leave it for now but am open to more discussion/opinions from the team.

Reply:
To @helronds's point, if we do add the `--in-place` argument, the change should be made in PR #47 since this tox testenv is going away and the arguments for autopep8 will be moved to the pre-commit configs in `.pre-commit-config.yaml`.